server/util/body-parser.js

```diff
@@ -157,15 +157,15 @@ function splitBlockquote(blockquote) {
       const body = `${wrapper.open}${quote}${wrapper.close}`;
       const footer = citation.length > 1 ? `<footer class="quote__footer"><cite class="quote__cite">${citation}</cite></footer>` : null;
 
-      return { body, footer };
+      return { body, footer, quote, citation };
     } catch (err) {
       return {
-        body: blockquote
+        body: blockquote, quote: blockquote
```

Comment:
Yea, I was going to mention that the parser shouldn't really have HTML in it, but hopefully it will be taken round the back of the shed soon.